### DISCUSSION TITLE
install it from github into HA

### DISCUSSION THREAD

**Discussion by E-driver6363** [syssi/homeassistant-goecharger-mqtt#57] [General]
Hi,
I´m sorry for this questestion, but I don´t know whats the link to install your homeassistant-goecharger-mqtt into the HA!!!

I tried to open via "..." ---- than Add-on Repositorys  --- but what link do I need than??

THANK you!

**Comment by syssi** [comment]
There is a difference between "add-ons" (= additional docker containers / 3rd party software) and custom components / integrations (= HA core compatible integrations which aren't part of the core). I recommend to install the "Home Assistant Community Stoer" (https://hacs.xyz/). This custom component can be found and installed via HACS.
